src/platforms/node/guides/aws-lambda/layer/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: AWS Lambda Layer (Node)
+description: "Learn how to set up Sentry with a Layer"
+---
+
+An alternative way of installing Sentry is using a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html). This happens instead of adding `@sentry/serverless` with NPM and Yarn. Your can still import Sentry as usual:
+
+```javascript
+const Sentry = require("@sentry/serverless");
+```
+
+<Alert level="warning">
+
+Note, this approach will not work if you use Webpack.
+
+</Alert>
```

Comment:
Let's clarify this -because it's either a Note or a Warning. Generally, we're moving away from colorful boxes that are more distracting than useful. In addition, I'm not quite clear on what we'd like to say - are we telling people not to use this approach if they're also using Webpack? In that case, perhaps:
```suggestion
<Note>

This installation method will not work if you are using Webpack.

</Note>
```